base/utils/openapi.go

```diff
@@ -12,3 +12,7 @@ func PtrTimeParse(ts string) *time.Time {
 	t, _ := time.Parse(time.RFC3339, ts)
 	return &t
 }
+
+func PtrString(v string) *string {
+	return &v
+}
```

Comment:
there's already vmaas.PtrString() in vmass API client

Reply:
ok, removed